airbyte-integrations/connectors/source-marketo/integration_tests/integration_test.py

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+#
+
+import pytest
+import json
+from typing import Mapping, Any
+
+from airbyte_cdk import AirbyteLogger
+from source_marketo.source import SourceMarketo
+from airbyte_cdk.sources.utils.schema_helpers import check_config_against_spec_or_exit
+
+CONFIG_FILE = "secrets/config.json"
+LOGGER = AirbyteLogger()
+
+
+class TestIntegrationMarketo:
+    @staticmethod
+    def get_config_file(config_path: str) -> Mapping[str, Any]:
```

Comment:
very small comment, but probably the right convention here is to use pytest fixtures

Reply:
Updated